pkg/bucket/lifecycle/lifecycle.go

```diff
@@ -213,7 +213,7 @@ type ObjectOpts struct {
 	VersionID        string
 	IsLatest         bool
 	DeleteMarker     bool
-	NumVersions      int
+	ParallelVersions int
```

Comment:
Still not sure what "Parallel" refers to?

Reply:
> Still not sure what "Parallel" refers to?

I replaced with NumVersions now.. it means how many other versions the current object has. This is needed for evaluating some lifecycle rules.